src/js/components/DropButton/stories/Simple.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { Close } from 'grommet-icons';
+
+import { Grommet, Box, Button, DropButton, Heading, Text } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const DropContent = ({ onClose }) => (
+  <Box pad="small">
+    <Box direction="row" justify="between" align="center">
+      <Heading level={3} margin="small">
+        Heading
+      </Heading>
+      <Button icon={<Close />} onClick={onClose} />
+    </Box>
+    <Text>Content</Text>
+  </Box>
+);
+
+DropContent.propTypes = {
+  onClose: PropTypes.func.isRequired,
+};
+
+const SimpleDropButton = () => {
+  const [open, setOpen] = React.useState();
+  const onClose = () => {
+    setOpen(false);
+    setTimeout(() => setOpen(undefined), 1);
```

Comment:
why do we need this?

Reply:
Otherwise open will always be false and the Drop won't open/show.

Reply:
this si very confusing since nobody will use it in real code.
how about we remove this line and create
```
const onOpen = () => {
    setOpen(true);
  };
```
 and then pass this function to the onOpen prop of the dropbutton? will match better the use case of people i think!